setup.py

```diff
@@ -58,9 +66,10 @@ def finalize_package_data(self):
             'Operating System :: OS Independent',
             'Programming Language :: Python :: 2 :: Only',
```

Comment:
I guess you can just change it to `Programming Language :: Python :: 2.7` and add some more with Python 3.* versions